### DISCUSSION TITLE
Unknown objects that appear during game play

### DISCUSSION THREAD

**Discussion by RQNG** [ppy/osu#26226] [Q&A] [answered]
What is the arrow pointing towards in the screenshot? It is not included in the skin file, so I don't know how to remove it. If this is a new version feature, please tell me how to remove it, as it greatly affects map reading.
<img width="319" alt="Snipaste_2023-12-29_21-44-22" src="https://github.com/ppy/osu/assets/115459678/198e7bb3-1948-4993-b028-cbfd42855baf">

**Comment by nana123126** [comment, marked as answer]
#26117
Perhaps adding empty elements can solve the problem.

**Reply by RQNG** [reply]
When I encountered this problem, I didn't know what to search for, but now the problem has been resolved. Thank you.
